src/auth-restrict.md

```diff
@@ -0,0 +1,302 @@
+
+Guacamole extensions are self-contained `.jar` files which are located within
+the `GUACAMOLE_HOME/extensions` directory. To install the
+guacamole-auth-restrict extension, you must:
+
+1. Create the `GUACAMOLE_HOME/extensions` directory, if it does not already
+   exist.
+
+2. Copy `guacamole-auth-restrict-1.6.0.jar` within `GUACAMOLE_HOME/extensions`.
+
+3. [As noted below](using-auth-restrict), apply additional restrictions to 
+   users, user groups, connections, and/or connection groups.
+
+:::{important}
+You will need to restart Guacamole by restarting your servlet container in
+order to complete the installation. Doing this will disconnect all active
+users, so be sure that it is safe to do so prior to attempting installation.
+:::
+
+(auth-restrict-config)=
+
+### Configuring Guacamole for advanced restrictions
+
+There are no guacamole.properties parameters that need to be configured
+in order to begin using the auth-restrict extension.
+
+:::{important}
+However, because the extension is capable of restricting access to logins
+and connections based on the client's IP address, it is important to make
+sure that Guacamole is receiving the correct IP address(es) for clients.
+This is particularly noteworthy when Guacamole is behind a reverse proxy,
+as noted in the [proxying Guacamole](reverse-proxy) manual page, but
+administrators should validate any potential network component that may
+block or adjust the IP address that Guacamole receives for client
+connections.
+:::
+
+(completing-auth-restrict-install)=
+
+### Completing the installation
+
+Guacamole will only reread `guacamole.properties` and load newly-installed
+extensions during startup, so your servlet container will need to be restarted
+before installation of the auth-restrict extension will take effect.
+
+(using-auth-restrict)=
+
+Using the auth-restrict extension
+---------------------------------
+
+Once the extension is installed and the web application restarted, an additional
+section of options will appear in the administrative pages of Guacamole that
+will allow administrators to configure the additional restrictions for various
+objects within the Guacamole system. The new section of options looks like this:
+
+![](images/auth-restrict-options.png)
+
+The next sections will cover how each restriction impacts the various objects
+in the system.
+
+### User Logins
+
+Additional settings can be applied to users and user groups that will restrict
+the ability of those individual users or members of the groups to log in to
+Guacamole.
+
+#### Restricting logins based on day and time
+
+This extension introduces the ability to restrict logins to the Guacamole
+system based on the day of the week and the time of day, and to combine these
+restrictions in a way that allows for flexibility in those schedules.
+
+As an example, let's say that you have a user or group of users whose access
+you'd like to restrict to your normal working hours of 9 AM to 5 PM, any day
```

Comment:
It might be worth being clear about what timezone these restrictions are in - UTC? The timezone of the admin editing the page? The configured server time? The user's timezone?

Reply:
Ah, yes, you're absolutely right - I knew I was forgetting something.

Reply:
Okay, added a blurb on that - let me know how it looks. Timezones and Daylight Savings Time get to be so much fun...

Reply:
Nice, looks good.